installer/test/CreateInstallerTestPackages/appxmanifest_ddlm_x64.xml

```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  xmlns:uap5="http://schemas.microsoft.com/appx/manifest/uap/windows10/5"
+  xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap uap3 uap5 com rescap">
+
+  <Identity
+    Name="ProjectReunion.Test.DDLM-x64"
```

Comment:
SUGGESTION: Use more specific package names to distinguish your test packages from others to avoid accidental collision (especially when investigating test failures), e.g. `Name="ProjectReunion.Installer.Test.DDLM-x64"`

Same comment applies throughout

Reply:
I distinguished names slightly differently but did add in the "Installer" keyword to make them more distinct and avoid accidental collisions and easier troubleshooting.